src/controllers/playback/video/index.js

```diff
@@ -209,8 +209,13 @@ import { setBackdropTransparency, TRANSPARENCY_LEVEL } from '../../../components
             if (!itemName) {
                 itemName = parentName || '';
             }
-
-            LibraryMenu.setTitle(itemName);
+            //Display the item with its premiere date if it has one
```

Comment:
```suggestion

            // Display the item with its premiere date if it has one
```

Reply:
I don't know where to put this comment, but at least add a blank line before and a space after `//`.

Reply:
Oops, I meant to add that as well.